medusa/server/web/home/handler.py

```diff
@@ -1222,6 +1222,27 @@ def sceneExceptions(show):
             out.append('S{season}: {names}'.format(season=season, names=', '.join(names)))
         return '<br>'.join(out)
 
+    @staticmethod
+    def check_show_for_language(show_obj, language):
+        """Request the show in a specific language from the indexer.
+
+        If the indexer throws the ShowNotFoundInLanguage Exception, we catch it,
+        and report it's not available in this language.
+        :param show_obj: (TVShow) Show object.
+        :param language: Language passed as string as a two letter country ccde. For ex: 'en'.
+
+        :returns: True (show found in language) False (show not found in language)
+        """
+        indexer_api_params = app.indexerApi(show_obj.indexer).api_params.copy()
+        indexer_api_params['language'] = language
+        indexer_api_params['episodes'] = False
```

Comment:
To give you guys a small update. I've changed the way the episode download is triggered. Previously it was passed as a param, but always passed as True. Meaning for every show search, you had no choice other then to always also download the episodes. Now the param config['episodes_enabled'] can be set, to toggle it.